config/pipelines/pacbio.yml

```diff
@@ -351,6 +351,8 @@ default: &default
           usedByType:
             type: :self
             value: publish_data&.used_by_type
+      avro_schema_version_2:
+        <<: *avro_schema_version_1
```

Comment:
One doubt, Is the latest avro_schema_version is v '2' or '3'?

Reply:
I think it's 2. Previously it was 1.